cupy/cuda/runtime.pyx

```diff
@@ -50,6 +50,7 @@ cdef extern from "cupy_cuda.h" nogil:
     # Error handling
     const char* cudaGetErrorName(Error error)
     const char* cudaGetErrorString(Error error)
+    int cudaGetLastError()
```

Comment:
It needs to be defined in `cupy_cuda.h`.

Reply:
Thanks!